docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -2,88 +2,180 @@
 
 You can use Bluetooth functionalities in your application, such as managing the local Bluetooth adapter, bonding, and exchanging data between Bluetooth-enabled devices. The Bluetooth standard provides a peer-to-peer (P2P) data exchange functionality over short distance between compliant devices.
 
-This feature is supported in mobile and wearable applications only.
+## Prerequisites
 
-The main features of the Bluetooth API include:
+To use the Application (in [mobile](../../api/latest/device_api/mobile/tizen/application.html), [wearable](../../api/latest/device_api/wearable/tizen/application.html) and [tv](../../api/latest/device_api/tv/tizen/application.html applications) and Bluetooth (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html) applications) APIs, the application has to request permission by adding the following privileges to the `config.xml` file:
 
-- Managing the local Bluetooth adapter   
+```
+<tizen:privilege name="http://tizen.org/privilege/application.launch"/>
+<tizen:privilege name="http://tizen.org/privilege/bluetooth"/>
+```
+## Handling UUIDs and binary data in Bluetooth API
+
+### Handling UUIDs
+According to the [Bluetooth Core Specification](https://www.bluetooth.com/specifications/bluetooth-core-specification/), UUIDs used to represent Bluetooth objects can take 3 forms:
+
+   * 128-bit representation: "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX", where each 'X' stands for a hexadecimal digit.  
+   For example: "198d3a9c-e21a-4f72-a48b-39a6bad7e583".
+   * 32-bit representation: "D<sub>1</sub>D<sub>2</sub>D<sub>3</sub>D<sub>4</sub>D<sub>5</sub>D<sub>6</sub>D<sub>7</sub>D<sub>8</sub>", equivalent to "D<sub>1</sub>D<sub>2</sub>D<sub>3</sub>D<sub>4</sub>D<sub>5</sub>D<sub>6</sub>D<sub>7</sub>D<sub>8</sub>-0000-1000-8000-00805F9B34FB", where D<sub>1</sub>..D<sub>8</sub> stand for hexadecimal digits.  
+   For example: "e72ad71b".
+   * 16-bit representation: "D<sub>1</sub>D<sub>2</sub>D<sub>3</sub>D<sub>4</sub>", equivalent to "0000D<sub>1</sub>D<sub>2</sub>D<sub>3</sub>D<sub>4</sub>-0000-1000-8000-00805F9B34FB", where D<sub>1</sub>..D<sub>4</sub> stand for hexadecimal digits.  
+   For example: "d182".
+
+Unless stated otherwise, each function taking a UUID as a parameter will accept any of the 3 forms and make appropriate conversions.  
+Unless stated otherwise, each UUID returned from a function and each UUID attribute of an object may be a 16-bit, 32-bit or 128-bit UUID.
+Bluetooth API functions are case-insensitive with regard to UUIDs - lowercase (abcdef) and uppercase (ABCDEF) characters are recognized as valid hexadecimal digits.
```

Comment:
> Request you to please rephrase to follow Tizen writing style.

Could you please give me a link to the Tizen writing style? I have tried to find it in this repository, but have only found the [markdown style guide ](https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md).
 
> Request you to please rephrase to follow Tizen writing style. Sentences like "Unless statd otherwise" is not recommended. For example,
> 
> Each function taking a UUID as a parameter accepts any of the three forms and makes appropriate conversions.
> Each UUID returned from a function and each UUID attribute of an object can be a 16-bit, 32-bit, or 128-bit UUID.

I am afraid, that rephrasing these sentences in this way would cause a loss of some, quite important, part of their meaning. We used the "Unless stated otherwise, ..." part to tell the reader, that some exceptions to these general rules are possible and they are stated in the documentation.

What do you think about adding the following sentence after them, to clarify, that there may be some exceptions to this rule?

> All exceptions to these rules are noted in the API reference for [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html) devices.

Reply:
I have applied the rest of your suggestions